src/app/modules/players/players.view.html

```diff
@@ -1,57 +1,82 @@
 <div class="container-fluid text-center">
-
-  <div class="col-xs-12">
-    <!-- TODO this string needs to be translated -->
-    <h2>seleziona giocatori</h2>
-  </div>
-
-  <div class="col-xs-12">
-    <div ng-show="vm.savedPlayers.getAll().length">
+<!--Title-->
```

Comment:
Il commento no è allineato con il tag successivo.
Puoi inoltre spaziare la parola interna in maniera da avere `<!-- SELECT PLAYERS -->`
Usa sempre upper case in modo tale che quando chiudi puoi avere `<!-- end of SELECT PLAYERS -->`